app/controllers/telegram_webhooks_controller.rb

```diff
@@ -97,4 +66,108 @@ def help(*)
   def action_missing(action)
     reply_with :message, text: "Can not perform #{action}" if command?
   end
-end
+
+  private
+  def getArgsFromCommand(* args)
+    #Day informed as a number
+    if args[0].match(NUMBER_REGULAR_EXPRESSION)
+
+      dayFromArgs = args[0].to_s.rjust(2, '0')
+      isCurrentMonth = true
+
+    elsif args[0].match(DATE_REGULAR_EXPRESSION)
+
+      dateSplitted = args[0].split('/')
+      dateFormated = Date.strptime(dateSplitted[0] + "/" + dateSplitted[1] + "/" + getCurrentYear, t(:date_format))
+      puts dateFormated.month
+      
+      dayFromArgs = dateFormated.strftime('%d')
+      monthFromArgs = dateFormated.strftime('%m')
+      isCurrentMonth = false
+
+    elsif args[0].is_a? String
+      #Day informed as a string (monday, tuesday...)
+      dayFromArgs = getDayFromString(args[0].to_s)
+      isCurrentMonth = true
+
+    else
+      #Command does not match anything
+      errorMessage = MESSAGE_CHECK_USAGE_COMMAND
+
+    end
+
+    if not errorMessage
+      if isCurrentMonth
+        errorMessage = validateDateFromInput(dayFromArgs.to_i, getCurrentMonth(nil).to_i)
```

Comment:
Sending nil as parameter seems a little bit odd to me (even though I got your idea). Wouldn't be better use Time.now.month?